app/controllers/mixins/start_url.rb

```diff
@@ -41,7 +36,6 @@ def start_page_options
 
   def start_page_allowed?(start_page)
     return false if STORAGE_START_PAGES.include?(start_page)
```

Comment:
This line looks like a bug.  Makes no sense to disallow unconditionally?
It used to also have ` && !::Settings.product.storage` condition before #204: https://github.com/simon3z/manageiq-ui-classic/commit/acf224afb3d4473f5271e8d87646506f02cf757b#diff-4c149c5e0e75ed63331744d901546b7cL44
cc @ZitaNemeckova @himdel